### PR TITLE
[Dy2Stat]Polish for zip in dy2stat

### PR DESCRIPTION
**Pull request by 0x45f**
### PR types

Others
### PR changes

Others
### Describe

polish for zip in dy2stat

### CODE REVIEW

File: python/paddle/fluid/dygraph/dygraph_to_static/call_transformer.py [hunk 1]
```diff
@@ -39,18 +39,19 @@ def _no_need_convert_call(self, node):
...
+          2. It's a python builtin function not include `len` and `zip`
         """
         assert isinstance(node, gast.Call)
         if is_paddle_api(node):
             return True
 
         func_str = ast_to_source_code(node.func).strip()
         try:
-            from paddle.fluid.dygraph.dygraph_to_static.convert_call_func import is_builtin_len, is_builtin
+            from paddle.fluid.dygraph.dygraph_to_static.convert_call_func import is_builtin_len, is_builtin, is_builtin_zip
```
Comment (Aurelius84) [inline python/paddle/fluid/dygraph/dygraph_to_static/call_transformer.py:50]: We should avoid `Runtime import`, please put it ahead of the file.

File: python/paddle/fluid/dygraph/dygraph_to_static/convert_call_func.py [hunk 2]
```diff
@@ -79,6 +79,12 @@ def is_builtin_len(func):
     return False
 
 
+def is_builtin_zip(func):
+    if is_builtin(func) and func.__name__ == 'zip':
+        return True
+    return False
```
Comment (Aurelius84) [inline python/paddle/fluid/dygraph/dygraph_to_static/convert_call_func.py]: `return is_builtin(func) and func.__name__ == 'zip'`

File: python/paddle/fluid/dygraph/dygraph_to_static/convert_operators.py [hunk 3]
```diff
@@ -298,6 +298,15 @@ def convert_len(var):
         return len(var)
 
 
+def convert_zip(*args):
+    for i, arg in enumerate(args):
+        if isinstance(arg, Variable) and arg.shape[0] == -1:
+            raise ValueError(
+                "If the parameter is Variable in 'zip', its first dimension should not be -1. "
+                "But args[{}].shape[0] == -1 in 'zip'".format(str(i)))
```
Comment (Aurelius84) [inline python/paddle/fluid/dygraph/dygraph_to_static/convert_operators.py]:
```suggestion
                Not support zip(tensor, ...) when tensor.shape[0] == -1, but found args[{}].shape[0] == -1 in 'zip'
```

File: python/paddle/fluid/dygraph/dygraph_to_static/convert_operators.py [hunk 4]
```diff
@@ -298,6 +298,15 @@ def convert_len(var):
         return len(var)
 
 
+def convert_zip(*args):
+    for i, arg in enumerate(args):
+        if isinstance(arg, Variable) and arg.shape[0] == -1:
+            raise ValueError(
```
Comment (Aurelius84) [inline python/paddle/fluid/dygraph/dygraph_to_static/convert_operators.py]: raise RuntimeError

File: python/paddle/fluid/dygraph/dygraph_to_static/call_transformer.py (hunk 1 above)
Comment (0x45f) [reply python/paddle/fluid/dygraph/dygraph_to_static/call_transformer.py:50]: 这里应该会有循环import的问题，放在前面会报错

File: python/paddle/fluid/dygraph/dygraph_to_static/convert_call_func.py (hunk 2 above)
Comment (0x45f) [reply python/paddle/fluid/dygraph/dygraph_to_static/convert_call_func.py]: Done

File: python/paddle/fluid/dygraph/dygraph_to_static/convert_operators.py (hunk 3 above)
Comment (0x45f) [reply python/paddle/fluid/dygraph/dygraph_to_static/convert_operators.py]: Done

File: python/paddle/fluid/dygraph/dygraph_to_static/convert_operators.py (hunk 4 above)
Comment (0x45f) [reply python/paddle/fluid/dygraph/dygraph_to_static/convert_operators.py]: Done

**Review by Aurelius84 — APPROVED** [review]

**Pull request merged**